### PR TITLE
Completed css-two-column-layout Challenge.

### PR DESCRIPTION
**Pull request by MohamadAlsyouf**
image

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+.p-margin {
+  margin-right: 47px;
+}
+
+.column-full {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 40px;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:76]: Only `width` or `flex-basis` on `column` classes, you should make an additional class you can add to the element in addition to the column class here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work on this! One note for you:

**Comment by MohamadAlsyouf** [comment]
image

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (MohamadAlsyouf) [reply css-two-column-layout/styles.css:76]: gotcha

**Review by codingcodymiller — APPROVED** [review]
Nice work on this!!

**Pull request merged**